packages/app/components/strategies/StrategyCard.tsx

```diff
@@ -39,6 +41,7 @@ export const StrategyCard = ({ strategy }: StrategyCardProps) => {
       onClick={() => {
         setSelectedStrategy(isSelected ? null : strategy);
         setShouldResetStackbox(isSelected);
+        trackClick(EVENTS.CLICK.STRATEGY_CARD);
```

Comment:
We should also track which strategy was clicked.

Reply:
I was thinking about that, which would be a nice way to track it?

Maybe using the token pair? Something like this maybe?
```ts
        trackClick(
          `${EVENTS.CLICK.STRATEGY_CARD} ${strategy.buyToken}/${strategy.sellToken}`
        );
```

WDYT?

Reply:
I think we can put it all, since there are strategies with the same pairs.

`${EVENTS.CLICK.STRATEGY_CARD}-${strategy.buyToken}/${strategy.sellToken}-${stategy.frequency}-${stategy.amount}`

Putting it all allows the person viewing the stats to know everything about the strategy easily.

But if you prefer just `amount` or `frequency` is enough to distinguish.

Reply:
I'll take this into Stackly channel so we can bring Venky into the discussion as well, so we're all on the same page :D (because I don't have strong opinions around this TBH)

Reply:
I think we can decide here tbh, just go to wtv you feel like.

Reply:
if we need more info we change it.

Reply:
Let's go with the complete one:

`${EVENTS.CLICK.STRATEGY_CARD}-${strategy.buyToken}/${strategy.sellToken}-${stategy.frequency}-${stategy.amount}`

We have the data there so let's expose it. Seeing that event makes it clear the strategy without any calc.

Reply:
I used this formula + conditioning the click event trigger only when the clicked strategy is NOT selected.